parameter_store.py

```diff
@@ -0,0 +1,7 @@
+import boto3
+
+def para_store(para_name):
+    ssm = boto3.client('ssm', region_name='us-east-2')
+    response = ssm.get_parameter(Name=para_name, WithDecryption=True)
+    return response['Parameter']['Value']
+
```

Comment:
please add if __name__ == "__main__" conditional statement to the file for its testing. Thanks.

Reply:
__ name __ == "__ main __"